allure-report-face/src/components/attachment/AttachmentView.hbs

```diff
@@ -38,6 +38,13 @@
     <div class="pane__section">
         <object class="attachment__embed" type="image/svg+xml" data="{{sourceUrl}}">Your browser does not support SVG</object>
     </div>
+{{else if (eq type "video")}}
+    <div class="pane__section">
```

Comment:
@just-boris just to double check, if it was a typo in your comment: did you mean `attachment__image` -> `attachment__media` renaming?.. or leave this class with the same name?